style.css

```diff
@@ -0,0 +1,292 @@
+  padding: 16px 16px 16px 16px;
+  background: #fff;
+}
+
+.menu-top a {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 16px;
+}
+
+.introduction h1 {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 900;
+  font-size: 40px;
+  line-height: 56px;
+  color: #172b4d;
+  padding: 96px 16px 0 16px;
+}
+
+.introduction p {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  color: #344563;
+  padding: 0 16px 16px 16px;
+}
+
+.link-up {
+  font-style: normal;
+  font-weight: 600;
+  color: #7f8cff;
+  padding: 0  16px;
+  font-size: 16px;
+}
+
+.social-media-icons {
+  display: flex;
+  padding: 24px 16px 50px 16px;
+}
+
+.social-media-icons li a {
+  margin-right: 32px;
+  font-size: larger;
+  color: #505f79;
+}
+
+.fab {
+  font-size: x-large;
+}
+
+main {
+  display: grid;
+  grid-template-columns: 100%;
+  row-gap: 5rem;
+  padding-top: 80px;
+}
+
+.work {
+  margin: auto;
+  max-width: 90%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  /* height: 38rem; */
+  background-color: #fff;
+  border-radius: 16px;
+  box-shadow: -0.2px -0.1px gray;
+}
+
+.img-work {
+  width: 90%;
+  height: auto;
+  padding-top: 16px;
+}
+
+.work h2 {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  color: #172b4d;
+  font-size: 40px;
+  margin: 8px;
+  align-self: flex-start;
+}
+
+.work-history {
+  display: flex;
+  align-items: center;
+  line-height: 20px;
+  margin: 16px 48px 16px 16px;
+  align-self: flex-start;
+}
+
+.work-history li {
+  padding-right: 8px;
+}
+
+.cover {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  color: #344563;
+  font-size: 14.4px;
+}
+
+.career {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 14.4px;
+  color: #7a869a;
+
+  /* padding-left: 0.5rem; */
+  text-transform: capitalize;
+}
+
+.body-detail {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  line-height: 25.6px;
+  color: #344563;
+  margin: 8px 16px 8px 14.4px;
+  font-size: 16px;
+}
+
+.courses {
+  display: flex;
+  align-self: flex-start;
+  padding: 16px 36px 0 16px;
+}
+
+.courses li {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  color: #6070ff;
+  border: 1px solid #dadbff;
+  border-radius: 0.5rem;
+  padding: 0 3.2px;
+  margin-right: 16px;
+  background-color: #ebebff;
+}
+
+.btn-body {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  color: #396df2;
+  margin: 16px 0 32px 16px;
+  text-transform: capitalize;
+  padding: 6.4px 8px;
+  border-radius: 8px;
+  background-color: #f7f7f9;
+  align-items: flex-start;
+  align-self: flex-start;
+}
+
+form {
+  align-self: flex-start;
+}
+
+.about-myself {
+  background-color: #fff;
+  border-radius: 0 80px 0 0;
+}
```

Comment:
- [ ] I believe this is the class used for the about section. The content of the section should be aligned with flexbox.